packages/communication-react/review/communication-react.api.md

```diff
@@ -1942,6 +1942,7 @@ export type MessageThreadProps = {
     disableEditing?: boolean;
     strings?: Partial<MessageThreadStrings>;
     fileDownloadHandler?: FileDownloadHandler;
+    dateTimeFormat?: (messageDate: Date) => string;
```

Comment:
This prop name is misleading to me.
When I think of `date time format` I think of the string template used for formatting a date.
For example, `YY-MM-DD` etc.
However, this callback is supposed to return the formatted datetime string which is confusing to me.